content/post/asahi-nixos/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: The Asahi NixOS Experience
+description: My 1.5 year experience with Asahi NixOS
+date: 2025-01-17 10:11:00 +0800
+categories:
+    - asahi-linux
+    - nix
+---
+
+In August of 2023, I was looking around for a new laptop. At the time,
+I was running an Alienware laptop with an RTX 3070 in it. However, it
+is a cumbersome machine with horrible battery life. The great thing was
+its performance. I needed a machine which was powerful, can run Linux,
+has great battery life, and portable. I considered x86 off the table as
+sleep modes on laptops has been a problem due to vendors not implementing
+them right. This led me to an ARM machine. Ultimately, I ended up with
+an Apple Macbook Pro from 2021 with an M1 Pro chip. It has 16GB of RAM and
```

Comment:
Line 16-17 tweak phrasing: "The introduction of compelling ARM-powered laptops led me to select the 2021 MacBook Pro..."

Reply:
Changed